.github/workflows/qe-image.yml

```diff
@@ -0,0 +1,47 @@
+name: Push qe images for new tag
+on:
+  push:
+    tags:
+      - '*'
+env:
+  IMAGE_REGISTRY: quay.io
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        go:
+          - '1.21'
+        os: ['linux', 'windows', 'darwin']
+        arch: ['amd64', 'arm64']
+        exclude:
+          - arch: 'arm64'
+            os: 'windows'
+          - arch: 'arm64'
+            os: 'linux'
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Log in to Quay.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ secrets.QUAY_IO_USERNAME }}
+          password: ${{ secrets.QUAY_IO_PASSWORD }}
+          registry: ${{ env.IMAGE_REGISTRY }}
+      - name: create script of building and push images
+        run: |
```

Comment:
I would simplify this with only one step and a script like

```bash
!#/bin/bash

 set -x
OS=${{matrix.os}} ARCH=${{matrix.arch}} CRC_INTEGRATION_IMG_VERSION=${{ github.ref_name }} make containerized_integration
podman push quay.io/crcont/crc-integration:${{ github.ref_name }}-${{matrix.os}}-${{matrix.arch}}
OS=${{matrix.os}} ARCH=${{matrix.arch}} CRC_E2E_IMG_VERSION=${{ github.ref_name }} make containerized_e2e
podman push quay.io/crcont/crc-e2e:${{ github.ref_name }}-${{matrix.os}}-${{matrix.arch}}
```

Reply:
updated